models/hugging_face/bert-base-cased/spec.yaml

```diff
@@ -0,0 +1,14 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+
+description: View model card at [HuggingFace](https://huggingface.co/bert-base-cased)
```

Comment:
>View model card at [HuggingFace](https://huggingface.co/bert-base-cased) [](http://example.com/codeflow?start=13&length=72)

Could we please update these?

Reply:
I am wondering if we should have a separate markdown so that it can be independently reviews.

Reply:
done